src/products/components/ProductStocks/ProductStocks.tsx

```diff
@@ -191,32 +191,40 @@ const ProductStocks: React.FC<ProductStocksProps> = ({
             {hasVariants ? (
               <>
                 <FormattedMessage
-                  defaultMessage="There are no warehouses set up for your store. To add stock quantity to the variant please"
+                  defaultMessage="There are no warehouses set up for your store. To add stock quantity to the variant please {configureWarehouseUrl}"
                   description="no warehouses info"
                   id="productVariantWarehouseSectionDescription"
-                />{" "}
-                <Link onClick={onWarehouseConfigure}>
-                  <FormattedMessage
-                    defaultMessage="configure a warehouse"
-                    description="no warehouses info"
-                    id="productVariantWarehouseSectionDescriptionLink"
-                  />
-                </Link>
+                  values={{
+                    configureWarehouseUrl: (
+                      <Link onClick={onWarehouseConfigure}>
+                        <FormattedMessage
+                          defaultMessage="configure a warehouse"
+                          description="no warehouses info"
+                          id="productVariantWarehouseSectionDescriptionLink"
+                        />
+                      </Link>
+                    )
+                  }}
+                />
```

Comment:
Okay I guess I need to be more precise:
https://formatjs.io/docs/react-intl/components/#rich-text-formatting

So I'd expect something like this:
`defaultMessage="There are no warehouses set up for your store. To add stock quantity to the variant please <link>configure a warehouse</link>"`